src/pages/Product.js

```diff
@@ -94,64 +69,31 @@ const Product = () => {
           <div className="relative flex flex-col w-full mt-16 lg:flex-row">
             <div className="relative lg:w-1/2 space-y-3 lg:space-y-5 2xl:pr-20">
               <div className="relative flex flex-col justify-center">
-                <button
-                  onClick={token ? () => setDownload(true) : () => redirect()}
-                  className="mb-6 py-3.5 hover-scale text-xl font-semibold leading-6 text-white whitespace-nowrap bg-primary rounded-md tracking-add shadow-button"
+                <a
+                  href={`${PUBLIC_URL}${product?.url}`}
```

Comment:
I have tried and this href is not working, it is redirecting me to:

![imagen](https://user-images.githubusercontent.com/66505715/155749699-308778c9-c055-48c1-b181-89e7d6636450.png)

I think they should be only the URL, some like:

```suggestion
                  href={`${product?.url}`}
```

Reply:
I fixed, plase check on #53